tile-reduce/index.d.ts

```diff
@@ -0,0 +1,139 @@
+// Type definitions for tile-reduce 3.1.1
+// Project: https://github.com/mapbox/tile-reduce
+// Definitions by: Denis Carriere <https://github.com/DenisCarriere>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace NodeJS  {
+  interface Global {
+    mapOptions: any
```

Comment:
Having an issue with this one, especially when "node" is imported as a definition, declaring `global` does work, but it immediately causes an error when you import the `node` definition.
Any suggestions?
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/node/node.d.ts